<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ez Namer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/Icon32x32.png</normaloff>Resources/Icon32x32.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_files_apply">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>528</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Re-Name Selected Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_index">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>484</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Index:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_extension">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>484</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Extension:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_files_deselect">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>400</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deselect All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_files_select_all">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>400</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_filter">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>47</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search Filter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>440</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_directory">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>20</y>
      <width>193</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_directory">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>12</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Directory:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dir_change">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_files">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>92</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Files:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>100</y>
      <width>353</width>
      <height>281</height>
     </rect>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_filter">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>56</y>
      <width>193</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>448</y>
      <width>257</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_index">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>488</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_extension">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>488</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dir_apply">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_filter_apply">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>56</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_files_refresh">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>400</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Refresh List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>16</number>
  </property>
  <property name="gridDeltaY">
   <number>4</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
